src/BTrees/check.py

```diff
@@ -11,6 +11,7 @@
 # FOR A PARTICULAR PURPOSE
 #
 ##############################################################################
+# isort: skip_file
```

Comment:
Was this file unfixable? Just wondering.

Reply:
I did not want isort to touch the groups of imports having a comment each. There is no way to have comments between imports in isort, if I understood correctly.